frontend/appflowy_tauri/package.json

```diff
@@ -7,24 +7,33 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "preview": "vite preview",
-    "tauri": "tauri"
+    "tauri:dev": "tauri dev"
```

Comment:
Also modify the `[tasks.tauri_dev]`

`npm run tauri dev` -> `npm run tauri:dev`